setup.py

```diff
@@ -12,9 +12,9 @@
     author='Harry Winters',
     author_email='harry.winters@twoporeguys.com',
     install_requires=[
-        'requests==2.19.1',
-        'sanic==0.7.0',
+        'requests==2.20.1',
```

Comment:
Why all the bumps here? Seems like it risks breakage unnecessarily. Also, do we still need `schematics` with the validators gone?

Reply:
oops, schematics should go right now.

I mean this is still not a `prod` release and we do have tests, so was wondering that I should take the chance and update them now before we enter more serious prod cycles.

I'll nuke the schematics req... good catch